nestjs/src/discord-servers/dto/discord-server.dto.ts

```diff
@@ -2,13 +2,16 @@ import { DiscordServer } from '@entities/discordServer';
 import { ApiProperty } from '@nestjs/swagger';
 
 export class DiscordServerDto {
-  constructor(discordServer: DiscordServer) {
+  constructor(discordServer: DiscordServer, reducedInfo: boolean = false) {
     this.id = discordServer.id;
-    this.createdDate = discordServer.createdDate;
-    this.updatedDate = discordServer.updatedDate;
     this.name = discordServer.name;
-    this.gratitudeUrl = discordServer.gratitudeUrl;
-    this.mentorsChatUrl = discordServer.mentorsChatUrl;
+
+    if (!reducedInfo) {
+      this.createdDate = discordServer.createdDate;
+      this.updatedDate = discordServer.updatedDate;
+      this.gratitudeUrl = discordServer.gratitudeUrl;
+      this.mentorsChatUrl = discordServer.mentorsChatUrl;
+    }
```

Comment:
If you'll use fullInfo flag, you won't need this logical negation
```suggestion
    if (fullInfo) {
      this.createdDate = discordServer.createdDate;
      this.updatedDate = discordServer.updatedDate;
      this.gratitudeUrl = discordServer.gratitudeUrl;
      this.mentorsChatUrl = discordServer.mentorsChatUrl;
    }
```